test/suites/standard1/kustomize.robot

```diff
@@ -267,6 +267,7 @@ Restore Default Config
     ${is_ostree}=    Is System OSTree
     IF    ${is_ostree}
         Reboot MicroShift Host
+        Wait For Greenboot Health Check To Exit
         Wait For MicroShift
```

Comment:
No need to Wait for MicroShift if we're checking Greenboot exit.

Reply:
thanks, addressed in the last commit